packages/react-forms/src/InputText/index.js

```diff
@@ -120,18 +120,16 @@ const Container = styled.div`
 
 const InputText: React.StatelessFunctionalComponent<Props> = styled(
   React.forwardRef(
-    ({ onChange, validationErrorMessage, as, ...props }: Props, ref) => {
+    (
+      { onChange, validationErrorMessage, as, renderAddon, ...props }: Props,
+      ref
+    ) => {
       const input = React.createRef();
       return (
         <InvalidHandler errorMessage={validationErrorMessage}>
           {(getInputProps, isInvalid) => (
             <ClassNames>
-              {({
-                css,
-                marginRightClass = css({
```

Comment:
Unfortunately this is an invalid signature for Emotion, I opened a PR (https://github.com/emotion-js/emotion/pull/1324) but there's some disagreement on it.